doc/source/install.txt

```diff
@@ -1,53 +1,153 @@
-Pre-built installation
+Installing scikit-image
+-----------------------
+If you are on Mac OS X you're lucky, open the terminal and install
```

Comment:
Because wheels exists and installation is straight forward containing only one step.